contribs/gmf/src/options.js

```diff
@@ -465,7 +465,7 @@
 
 /**
  * @typedef {Object} gmfEditFeatureOptions
- * @property {import('ngeo/options.js').StyleLike} highlightStyle A style object for highlighting of feature.
+ * @property {import('ngeo/options.js').Style} highlightStyle A style object for highlighting of feature.
```

Comment:
What is this change for ? OL use StyleLike (https://openlayers.org/en/latest/apidoc/module-ol_style_Style.html#~StyleLike) and there is 9 others occurance of it in this same file.

Reply:
+1 it should be `StyleLike`